torch/serialization.py

```diff
@@ -1268,15 +1272,16 @@ def persistent_load(saved_id):
 
```

Comment:
maybe add a comment to `result = unpickler.load()` with something like `create tensors with uninitialized data`?

It would also be great to have a comment explaining why we need both `pickle_module.load(f, **pickle_load_args)` AND `result = unpickler.load()`.